.github/workflows/auto-update-community.yaml

```diff
@@ -127,6 +127,9 @@ jobs:
         body: |
           ${{ github.event.inputs.reason || 'Cron' }} -${{ github.actor }}
 
+          /cc ${{ matrix.assignees }}
```

Comment:
Dont think we want these lines as updating community files is done by Knative Automation without intervention from people. This would just cause more spam.

Reply:
Ah! extremely sorry. Missed the real context because of the issue title. Fill fix it ASAP

Reply:
No worries i think this is a non issue tbh since we have not had any requests for the log to be placed back in.

Reply:
So, the original issue is poorly written. I apologize. Basically we are getting pinged (via email) when one of the automated PRs that get auto merged. It is spammy because we don't need to be reveiwing a PR that has a skip-review. If you can make it to the productivity WG meeting we can talk more and see other work that might be more impactful. If the time is not good then we can try to meet up outside of that please feel free to reach out via CNCF slack.

Reply:
Updated the issue to capture more details and hopefully make things clearer.